app/gui/src/main/resources/themes/dark-theme.css

```diff
@@ -66,11 +66,23 @@
     -fx-border-width: 1px;
 }
 
-/* Accordion Group heading content */
+/* Accordion TitledPane heading content */
 .small {
     -fx-font-size: 0.8em;
 }
 
+/* Accordion TitledPane heading textfield */
+.title-textfield {
+    -fx-pref-width: 90;
+}
+
+/* Adjust divider visibility and padding */
+/*
+.split-pane > .split-pane-divider {
+    -fx-padding: 0;
+}
```

Comment:
deletable?

Reply:
I would leave this one in case we want to play with it. It took some googling to find.